config/development.json

```diff
@@ -43,6 +43,7 @@
 		"dev/test-helper": true,
 		"guided-tours": true,
 		"guided-tours/main": true,
+		"guided-tours/theme": true,
```

Comment:
I'm going to change this too to `theme-sheet-welcome` and enable it in `wpcalypso` as well.